src/main/java/com/quantumlytangled/gravekeeper/GraveKeeperConfig.java

```diff
@@ -218,9 +213,11 @@ public static void loadConfig(final File file) {
         .get("soulbound", "soulbound_enchantment_names", SOULBOUND_ENCHANTMENT_NAMES,
             "List of enchantment names that are considered as soulbinding")
         .getStringList();
-    SOULBOUND_TAG_VALUE = config
-        .get("soulbound", "soulbound_tag_value", SOULBOUND_TAG_VALUE,
-            "List of NBT values that are considered as soulbinding when it equals the value after the \":\"")
+    SOULBOUND_TAG_STRINGS = config
+        .get("soulbound", "soulbound_tag_strings", SOULBOUND_TAG_STRINGS, String.join("\n", new String[]{
```

Comment:
the configuration key has no reason to say strings, consider renaming soulbound_tag_strings to soulbound_tags